eth_defi/hsm_hotwallet.py

```diff
@@ -0,0 +1,474 @@
+"""HSM wallet management utilities.
```

Comment:
You need to add new files to `docs/source` indexes, as currently Sphinx cannot pick up all stuff automatically.